content/10.methods.md

```diff
@@ -118,6 +118,7 @@ Of 34,005 corresponding authors, 45 were missing a fore name altogether in the r
 Of the remaining authors, genderize.io failed to predict gender for 1,578 of these fore names.
 We note that approximately 52% of these NA predictions are hyphenated names, which is likely because they are more unique and thus are more difficult to find predictions for.
 87% of these names were predicted to be of Asian origin by last name (see the race/ethnicity prediction model below).
+While this bias of NA predictions toward non-English names has been previously observed [@doi:10.32614/RJ-2016-002], it did not affect the accuracy of the gender classification and the corresponding conclusion.
```

Comment:
Do you mean in our paper or the one you are citing? If our paper, how do you know?

Reply:
The one we're citing. 

But in our paper: 
> 87% of these names were predicted to be of Asian origin by last name (see the race/ethnicity prediction model below).

Reply:
The way the sentence is structured, I couldn't tell if you were saying "it did not affect the accuracy of the gender classification and the corresponding conclusion" for our work or them. If us, I didn't understand how that was known to be true. If them, I'd just move the citation to the end.

Reply:
I see what you mean now. And you're right. We don't.

What I meant was that this bias toward non-English names, different from, say, bias toward female names, is not directly affecting our estimate of gender compositions. It is difficult to quantify or even figure out which direction the bias would affect the final estimation (over vs. under-estimate of female scienists) because we don't have the ground truth.

How about this:
```suggestion
This bias of NA predictions toward non-English names has been previously observed [@doi:10.32614/RJ-2016-002] and may have a minor influence on the final estimate of gender compositions.
```

Reply:
Works for me 👍